srunner/scenarios/actor_flow.py

```diff
@@ -319,3 +328,93 @@ def __del__(self):
         Remove all actors and traffic lights upon deletion
         """
         self.remove_all_actors()
+
+class CleanCrossActorFlow(BasicScenario):
+    """
+    This scenario is similar to CrossActorFlow
```

Comment:
Similar to the name, the docs should reflect the expected behavior of the scenario, along with its limitations